code/game/objects/items/stacks/stack.dm

```diff
@@ -52,14 +52,16 @@
 	update_icon(UPDATE_ICON_STATE)
 
 /obj/item/stack/update_icon_state()
-	. = ..()
```

Comment:
Is there any specific reason you removed this from the beginning? I think its cleaner to have it at the beginning